packages/origin-backend-core/src/OrganizationInvitation.ts

```diff
@@ -2,9 +2,10 @@ import { IOrganization } from './Organization';
 import { Role } from './User';
 
 export enum OrganizationInvitationStatus {
-    Pending,
     Rejected,
-    Accepted
+    Accepted,
+    Sent,
+    Viewed
```

Comment:
Do not reorder those enums, as existing records in DB will be incorrectly mapped:

Previous Pending = Rejected
Rejected = Accepted ....

So either append only or assign int numbers explicitly so they are not messed up.